src/panels/lovelace/cards/energy/hui-energy-gas-graph-card.ts

```diff
@@ -318,11 +334,21 @@ export class HuiEnergyGasGraphCard
 
   private _processDataSet(
     statistics: Statistics,
+    statisticsMetaData: StatisticsMetaData[],
     gasSources: GasSourceTypeEnergyPreference[],
     gasColor: string,
     compare = false
   ) {
     const data: ChartDataset<"bar", ScatterDataPoint[]>[] = [];
+
+    const getLabelForNoEntity = (stat_energy_from: string) => {
```

Comment:
Can we make this a generic helper function

Reply:
which file is good place to put generic helper function?

Reply:
`data/history.ts` for statistics.

Reply:
I put it in `energy.ts` as this applies only for energy stats I think. Let me know if you prefer `history.ts` anyway, then I can move it.

Reply:
I think this applies to every stat right?